lib/auto-languageclient.js

```diff
@@ -59,6 +59,10 @@ export default class AutoLanguageClient {
     throw Error('Must override startServerProcess to start language server process when extending AutoLanguageClient');
   }
 
+  newLinterPushV2Adapter(connection: ls.LanguageClientConnection): LinterPushV2Adapter {
```

Comment:
Given how many adapters we have and how many more to come I'm wondering (out loud) if we might be better off just having a single method do them all or some simple IoC/Dependency Injection pattern.